src/common/util.js

```diff
@@ -114,10 +114,8 @@ function getNextNonSpaceNonCommentCharacterIndex(text, node, locEnd) {
  * @returns {string}
  */
 function getNextNonSpaceNonCommentCharacter(text, node, locEnd) {
```

Comment:
This function is not a good util to export, why use call `locEnd(node)` themself? I'm going to make a change first.